packages/app-desktop/gui/note-viewer/index.html

```diff
@@ -104,23 +105,46 @@
 		// it at any time knowing that it's not going to be changed because the content height has changed.
 		let percentScroll_ = 0;
 
-		// This variable provides a	way to skip scroll events for a certain duration.
-		// In general, it should be set whenever the scroll value is set explicitely (programmatically)
+		let ignoreNextScrollTime_ = Date.now();
+		let ignoreNextScrollEventCount_ = 0;
+
+		// ignoreNextScrollEvent() provides a way to skip scroll events for a certain duration.
+		// In general, it should be called whenever the scroll value is set explicitely (programmatically)
 		// so as to differentiate scroll events generated by the user (when scrolling the view) and those
 		// generated by the application.
-		let lastScrollEventTime = 0;
+		function ignoreNextScrollEvent() {
+			const now = Date.now();
+			if (now >= ignoreNextScrollTime_) ignoreNextScrollEventCount_ = 0;
+			if (ignoreNextScrollEventCount_ < 10) { // for safety
+				ignoreNextScrollTime_ = now + 200;
+				ignoreNextScrollEventCount_ += 1;
+			}
+		};
```

Comment:
I think this behaviour is the same as in useScrollHandler.ts, do you think it would be possible to add this functionality into utils?

Reply:
That's right, the same. The following points let me not to merge them.
- It seems better to me to keep files in `note-viewer/` isolated from others.
- They just happen to match now, but the requirements from two different underlying layers (CodeMirror and <iframe>) are very sensitive. If the requirements change, they may not match anymore.